runner/app/pipelines/audio_to_text.py

```diff
@@ -20,7 +20,7 @@
 
 
 class AudioToTextPipeline(Pipeline):
-    def __init__(self, model_id: str):
+    def __init__(self, model_id: str, return_timestamps: str = "false"):
```

Comment:
@eliteprox variable appears to be unused.

Reply:
Removed unused variable